modules/perinatal/controller/carnet-perinatal.events.ts

```diff
@@ -25,11 +25,15 @@ EventCore.on('perinatal:control:validacion', async ({ prestacion, registro }) =>
             query['embarazo.conceptId'] = embarazo.conceptId;
         }
         const carnetExistente: any = await CarnetPerinatal.findOne(query);
+
+        const proxControl = prestacion.ejecucion.registros.find(itemRegistro => itemRegistro.concepto.conceptId === '2471000246107').valor;
```

Comment:
@MCele consulta: en vista de que en la linea siguiente se evalúa el valor de `proxControl`, existe la posibilidad de que la ejecución no contenga un registro con concepto `2471000246107`?
En tal caso, creo que habría que controlar la existencia en el find:
```suggestion
        const proxControl = prestacion.ejecucion.registros.find(itemRegistro => itemRegistro.concepto.conceptId === '2471000246107')?.valor;
```

Reply:
Qué buen ojo! La realidad es que actualemente en las prestacones que se registra el embarazo ese concepto ya está incluido, aunque "valor" quede en null. Pero de todos modos es necesario el cambio que proponés para dejarlo genérico.